src/SDKs/SqlManagement/Sql.Tests/ManagedDatabaseRestoreScenarioTests.cs

```diff
@@ -0,0 +1,89 @@
+﻿using Microsoft.Azure.Management.ResourceManager;
+using Microsoft.Azure.Management.ResourceManager.Models;
+using Microsoft.Azure.Management.Sql;
+using Microsoft.Azure.Management.Sql.Models;
+using Microsoft.Azure.Management.Network.Models;
+using Sku = Microsoft.Azure.Management.Sql.Models.Sku;
+using System;
+using System.Collections.Generic;
+using Xunit;
+
+
+namespace Sql.Tests
+{
+    // More tests will be added here once we start working on RestorableDroppedDatabase clients
+    //
+    public class ManagedDatabaseRestoreScenarioTests
+    {
+        [Fact]
+        public void ShortTermRetentionOnLiveDatabase()
+        {
+            SqlManagementTestContext Context = new SqlManagementTestContext(this);
+            ResourceGroup resourceGroup;
+            ManagedInstance managedInstance;
+            string testPrefix = "manageddatabaserestorescenariotest-";
+
+            try
+            {
+                SqlManagementClient sqlClient = Context.GetClient<SqlManagementClient>();
+                resourceGroup = Context.CreateResourceGroup();
+
+                // Create vnet and get the subnet id
+                VirtualNetwork vnet = ManagedInstanceTestFixture.CreateVirtualNetwork(Context, resourceGroup, TestEnvironmentUtilities.DefaultLocationId);
```

Comment:
It's awesome that you are creating all the requirements in the test setup, so the test is fully automated. Thank you for doing this. It looks a little different than how other tests are using ManagedInstanceTestFixture, please consider using the same approach so that the code is more consistent and quick to understand.

Reply:
I tried using ManagedInstanceTestFixture class, but had some issues. Since SqlManagementTestContext is initialized in TestFixture class it means that all SessionRecords using that class will go into "Sql.Tests.ManagedInstanceTestFixture". Because of this record for my tests would overwrite existing record, causing ManagedDatabaseVunerabilityAssesmentScenarioTests to fail in non-record mode, and I wanted to avoid that.